markdown/org/docs/patterns/shelly/options/sideshape/en.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Side shape"
+---
+
+Controls how the side seam is curved. Zero creates a straight line from the armpit to the side of the hips. Positive values add material and gives the side seam a rounded/convex shape, while negative values remove material and gives the side seam an hourglass/concave shape.
+
```

Comment:
Maybe also mention that this option has no effect when the `straight sides` option is enabled.